services/user.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib.auth import get_user_model
+from typing import Any
+
+
+User = get_user_model()
+
+
+def create_user(username: str, password: Any, email: Any = None,
+                first_name: str = None, last_name: str = None) -> None:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()

```

Comment:
You are calling the `save()` method multiple times in your function. This is not efficient and goes against the DRY (Don't Repeat Yourself) principle. You should set all the attributes first and then call the `save()` method only once. Here's an example of how you can do it:
```python
user = get_user_model().objects.create_user(username=username, password=password)
if email:
    user.email = email
if first_name:
    user.first_name = first_name
if last_name:
    user.last_name = last_name
user.save()
```
This is a code style issue.